### PR TITLE
feature notifications

### PR DESCRIPTION
**Pull request by theMetalkate**
Notifications
* highlight every second row for better reading and differentiation
* unreaded items are highlighted in bold and with a small dot indicator infront of the line

Screenshot 2020-03-29 at 14 29 58
Screenshot 2020-03-29 at 14 30 13

### CODE REVIEW

File: src/components/content/Notifications.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    timestamp: string,
+    readed: boolean
+}
+
+export default function Notifications(entries: NotificationProp[]) {
+    return (
+        <Wrapper>
+            {entries.map((entry, index) => {
+                return (
+                    <Item readed={entry.readed} className={ ( !entry.readed ? "unreaded" : "") }>
```
Comment (Entkenntnis) [inline src/components/content/Notifications.tsx]: you are already passing the prop to the styled component, consider removing the className and adapt the styling using this prop

File: src/components/content/Notifications.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        &:before {
+            content: "";
+            display: inline-block;
+            background: ${props => props.theme.colors.brand};
+            border-radius: 50%;
+            width: 10px;
+            height: 10px;
+            margin-right: 7px;
+        }
+    }
```
Comment (Entkenntnis) [inline src/components/content/Notifications.tsx]: you could write something like

```tsx
`
  ${props => props.unreaded ? `
      font-weight: ...;
      color: ${props.theme.colors.brand};
      ....;
    ` : ''}
`
```

Comment (theMetalkate) [reply src/components/content/Notifications.tsx]: Ok, didn't know this is working. Should be fine noe

File: src/components/content/Notifications.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+}
+
+interface NotificationProp {
+  author: User
+  body: string
+  timestamp: string
+  readed: boolean
+}
+
+export default function Notifications(entries: NotificationProp[]) {
```
Comment (Entkenntnis) [inline src/components/content/Notifications.tsx:16]: Hi Kate, thank you for your contribution! To make this file a proper react function component, the argument must be an object with entries as a key, so we need `Notifications({entries} : NotificationsProps)` here, and the interface should be adapted, too.

Could you please fix this? The rest looks fine.

Comment (theMetalkate) [reply src/components/content/Notifications.tsx:16]: fixed

**Pull request merged**